lstchain/reco/r0_to_dl1.py

```diff
@@ -411,7 +411,8 @@ def r0_to_dl1(
                 pulse_templates = {tel_id: NormalizedPulseTemplate.load_from_eventsource(
                     subarray.tel[tel_id].camera.readout, resample=True)
                     for tel_id in config['source_config']['LSTEventSource']['allowed_tels']}
-                if 'nsb_tuning_rate' in config['waveform_nsb_tuning'].keys():
+                if 'nsb_tuning_rate_GHz' in config[
+                    'waveform_nsb_tuning'].keys():
```

Comment:
.keys() is superfluous

Reply:
There are also other instances, will take care